custom/icds_reports/static/js/angular-services/baseControllers.service.js

```diff
@@ -60,6 +60,10 @@ window.angular.module('icdsApp').factory('baseControllersService', function() {
             vm.haveAccessToFeatures = haveAccessToFeatures;
             vm.message = storageService.getKey('message') || false;
 
+            // variables used for chart rendering. can be overridden by subclasses
+            vm.usePercentage = true;
+            vm.forceYAxisFromZero = false;
```

Comment:
this batch of reports overrode these two variables, so they got pulled up into the class